packages/biometric-ed25519/package.json

```diff
@@ -13,19 +13,19 @@
   "author": "Pagoda",
   "license": "ISC",
   "dependencies": {
-    "@aws-crypto/sha256-js": "^4.0.0",
-    "@hexagon/base64": "^1.1.26",
+    "@aws-crypto/sha256-js": "4.0.0",
+    "@hexagon/base64": "1.1.26",
     "@near-js/crypto": "workspace:*",
     "@near-js/utils": "workspace:*",
-    "asn1-parser": "^1.1.8",
+    "asn1-parser": "1.1.8",
     "bn.js": "5.2.1",
     "borsh": "1.0.0",
-    "buffer": "^6.0.3",
-    "elliptic": "^6.5.4",
+    "buffer": "6.0.3",
+    "elliptic": "6.5.4",
```

Comment:
why is elliptic used here in ed25519 context? elliptic's implementation of the curve is buggy and produces invalid outputs, which means in blockchain context "someone will lose money"